src/data/nomad.json

```
@@ -28,5 +29,37 @@
         "href": "https://support.hashicorp.com"
       }
     ]
-  }
+  },
+  "navigationHeaderItems": [
+    {
+      "label": "Overview",
+      "path": "/nomad",
+      "id": "overview"
+    },
+    {
+      "label": "Docs",
+      "id": "docs",
+      "path": "/nomad/docs"
+    },
+    {
+      "label": "API",
+      "path": "/nomad/api-docs",
+      "id": "api-docs"
+    },
+    {
+      "label": "Plugins",
+      "path": "/nomad/plugins",
+      "id": "commands"
+    },
+    {
+      "label": "Tools",
+      "path": "/nomad/tools",
+      "id": "commands"
+    },
```

These two have the same `id` specified, and it's making React unhappy because they're used as `key` props in a map.

```suggestion
      "label": "Plugins",
      "path": "/nomad/plugins",
      "id": "plugins"
    },
    {
      "label": "Tools",
      "path": "/nomad/tools",
      "id": "tools"
    },
```

Side note: I think these IDs probably aren't needed anymore, but let's address that outside of this PR. 😊

---

ah dang thanks for catching this, totally missed it 😅 Applied! ✅

---

All good! You and Bryce catch my copy-paste errors all the time 😁